runner/utils.go

```diff
@@ -25,12 +25,14 @@ func isTmpDir(path string) bool {
 
 func isIgnoredFolder(path string) bool {
 	paths := strings.Split(path, "/")
-	if len(paths) <= 0 {
+	pathsOnWindows := strings.Split(path, "\\")
```

Comment:
I think it's exactly the same but you use the filepath pkg instead of strings: [https://golang.org/pkg/path/filepath/#Split](https://golang.org/pkg/path/filepath/#Split)